src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -197,14 +204,34 @@ private boolean cleanSubscriptionRepeatFlagHandlerDb(String subscriptionNameQuer
      *            name of the current user
      * @return a String object
      */
-
     private String generateQuery(String subscriptionName, String ldapUserName) {
         String query = String.format(SUBSCRIPTION_NAME, subscriptionName);
-        if (!ldapUserName.isEmpty()) {
+        if (ldapUserName != null && !ldapUserName.isEmpty()) {
             String queryUser = String.format(USER_NAME, ldapUserName);
             String queryTemp = query + "," + queryUser;
             query = String.format(AND, queryTemp);
         }
         return query;
     }
+
+    /**
+     * This method finds whether a given subscription has an owner
+     *
+     * @param subscriptionName-
+     *            subscription name
+     * @return a boolean
+     * @throws SubscriptionNotFoundException
+     */
+    private boolean doSubscriptionOwnerExist(String subscriptionName) {
+        boolean ownerExist = false;
+        try {
+            if (!getSubscription(subscriptionName).getLdapUserName()
```

Comment:
I believe this if statement should be in one line, we do not use Christoffers formatter any more since it had few flaws. Use Standard Eiffel formatter.

Reply:
Anyway, I changed it as a one liner...